### PR TITLE
py-numpy: add last version 1.17.4

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-numpy/package.py [hunk 1]
```diff
@@ -26,7 +26,8 @@ class PyNumpy(PythonPackage):
         'numpy.distutils.command', 'numpy.distutils.fcompiler'
     ]
 
-    version('1.15.2', sha256='27a0d018f608a3fe34ac5e2b876f4c23c47e38295c47dd0775cc294cd2614bc1')
+    version('1.17.4', sha256='f58913e9227400f1395c7b800503ebfdb0772f1c33ff8cb4d6451c06cabdf316')
+    version('1.15.2', sha256='27a0d018f608a3fe34ac5e2b876f4c23c47e38295c47dd0775cc294cd2614bc1', Preferred=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-numpy/package.py]: why prefer the old one?

Comment (alkino) [reply var/spack/repos/builtin/packages/py-numpy/package.py]: due to recompilation, I guess?

Comment (alkino) [reply var/spack/repos/builtin/packages/py-numpy/package.py]: @pramodk Is it important for a python module?

**Comment by alkino** [comment]
retest this please Jenkins

**Comment by alkino** [comment]
retest this please Jenkins

**Comment by alkino** [comment]
numpy stop supporting python2 since 1.17.2 so we cannot upgrade it.

**Pull request closed without merging**